tools/filters/gff/gff_filter_by_feature_count.py

```diff
@@ -5,6 +5,8 @@
 Usage:
 %prog input_name output_name feature_name condition
 """
+from __future__ import print_function
```

Comment:
Actually the file is now using a mixed syntax, twice the old py2.7 one, once the new py3 one. I think it's broken!